sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -2,12 +2,15 @@
 
 ## 1.5.0 (Unreleased)
 
+
 ### Features Added
+- Added ability to configure new cloud environments in two different ways
+- Setting an environment variable `ARM_CLOUD_METADATA_URL`, the SDK will then follow that URL and look for cloud configurations there
+- Passing in keyword arguments from the command line when creating a new MLClient object
```

Comment:
Is this SDK specific? If yes can you please share an example on how this will work ?